x-pack/test/functional/apps/lens/group1/smokescreen.ts

```diff
@@ -280,13 +280,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       let data = await PageObjects.lens.getCurrentChartDebugState('xyVisChart');
-      expect(data?.axes?.y?.[0].title).to.eql(axisTitle);
+      expect(data?.axes?.y?.[1].title).to.eql(axisTitle);
 
       // hide the gridlines
       await testSubjects.click('lnsshowyLeftAxisGridlines');
 
       data = await PageObjects.lens.getCurrentChartDebugState('xyVisChart');
-      expect(data?.axes?.y?.[0].gridlines.length).to.eql(0);
+      expect(data?.axes?.y?.[1].gridlines.length).to.eql(0);
```

Comment:
Why has it changed from the first to the second y axis?

Reply:
Previously we have only two global axis (left and right) in that order in array. As this test depends on previous and in previous we switch to `right` axis and that create axis with id we should check second value from array. (axis with id goes first in the array)